source/joker.c

```diff
@@ -48,7 +50,7 @@ static bool used_layers[MAX_JOKER_OBJECTS] = {false}; // Track used layers for j
 
 // Maps the spritesheet index to the palette bank index allocated to it.
 // Spritesheets that were not allocated are
-static int* joker_spritesheet_pb_map;
+static int joker_spritesheet_pb_map[MAX_SPRITES];
```

Comment:
```suggestion
static int joker_spritesheet_pb_map[(MAX_DEFINABLE_JOKERS + 1)/NUM_JOKERS_PER_SPRITESHEET];
```

The size of this array is not really tied to the number of sprites that can be displayed, theoretically if we implement all 150 jokers and decide to decrease `NUM_JOKERS_PER_SPRITESHEET` to 1, it would be larger. I suggest defining MAX_DEFINABLE_JOKERS as 150 to get a better cap for the size of this array and avoid the malloc.